### DISCUSSION TITLE
Change of NZBGet Repo?

### DISCUSSION THREAD

**Discussion by DynaMight1124** [OneCDOnly/sherpa#282] [General]
Hi

Hopefully I'm not bugging you? Please say if I am! NZBGet hasnt been updated by the original developer for a long time and their Github was archived last year. I do still use NZBGet as my newsgroup client, I should probably use SABnzbd but I like the speed of NZBGet, although I have a faster NAS now, it was like 5 times faster when I had an old arm based QNAP years ago.   

I see theres a continuation of the app on https://github.com/nzbgetcom/nzbget, which has been getting updates and seemingly deemed as the 'official' replacement. I also saw theres a QNAP package pull request in progress.

Is there any thoughts about changing the repo from the original to this newer one?

Thanks

**Comment by OneCDOnly** [comment]
Hi mate. :)

Yes, I've been watching that one, and I've already included it in my list of potential applications to package: https://github.com/OneCDOnly/sherpa/blob/bbf0b29936b3d099ed09b1770eba643495bde40a/workshop/potential-packages/new-packages.txt#L25

At the moment, I'm concentrating on getting sherpa ready to finally remove the beta tag, but I'll be looking at adding a few new QPKGs after that. 👍🏽

**Reply by DynaMight1124** [reply]
Thats awesome, thank you

**Comment by OneCDOnly** [comment]
@DynaMight1124 , good news, sherpa will now manage a new NZBGet QPKG created by the new development team! 👍🏽 

To transfer to the new QPKG, please be sure to read the notes here first: https://github.com/OneCDOnly/sherpa/wiki/upgrading-NZBGet

**Reply by DynaMight1124** [reply]
Thats awesome, thanks. I followed the guide and all worked fine. 

I did backup my config via the WebUI and tried to restore (for science :) ), I think a lot of the restore was compatible, I have a few apps that communicate with NZBGet and when restored they could communicate with it however the WebUI didnt work. It looks like the main changes are around the directories, everything is now stored in NZBGet main dir, whereas it was stored in other locations previously, this broke the UI. 

I've compared the WebUI backed up configs between the old v21 and new v24 version, theres 2 new config lines (ArticleReadChunkSize=4 & Server1.CertVerification=Strict) all other config lines are the same with UI breaking changes to the below:
WebDir=${AppDir}/webui
LockFile=${MainDir}/nzbget.lock
LogFile=${MainDir}/nzbget.log
ConfigTemplate=${AppDir}/webui/nzbget.conf.template

If I get some free time later, I'll do a clean v24 install, manually change the 4 directory locations in my v21 backup config and restore that. It might be an easier user upgrade path.

Thanks for the amazing work you've done!

Thanks
